src/config/script/command/index.js

```diff
@@ -36,11 +45,47 @@ class Command {
 
   static copy(command) {
     if (!(command instanceof Command)) {
-      throwError(`Command.copy expects a Command instance but instead recieved a ${(command).constructor.name} instance`);
+      throwError(`Command.copy expects a Command instance but instead received a ${(command).constructor.name} instance`);
     }
     return lodash.cloneDeep(command);
   }
 
+  /**
+   * Prompts the user for values for the command variables and saves them to the command directives
+   *
+   * @returns Promise updatedDirectivesWithVariables - a promise that all the command directives have been updated with command variables
+   */
+  updateDirectivesWithVariables() {
+    const promises = [];
+
+    const variables = this.getVariables();
+
+    // Setup subscriber to receive the answers and save the directives
+    const subscriber = inquirerPrompts.subscribe(({ name: { variable, resolve }, answer }) => {
+      const variableRegex = new RegExp(`${variable}`, 'g');
+      const updatedDirectives = this.getDirectives().map(directive => directive.replace(variableRegex, answer));
+      this.updateDirectives(updatedDirectives);
+      resolve();
+    }, noop, noop);
+
+    Object.keys(variables).forEach((variable) => {
+      // Ask question
+      promises.push(new Promise((resolve) => {
+        prompts.next({
+          type: 'input',
+          name: {
+            variable,
+            resolve,
+          },
+          message: variables[variable],
+          default: '',
```

Comment:
Maybe this should change. However, there isn't validation for any commands yet, so I won't add any just yet.